build.gradle

```diff
@@ -25,7 +25,7 @@ buildscript {
         // App dependencies
         constraintLayoutVersion = '2.0.0-alpha3'
         coreTestingVersion = '2.0.0'
-        coroutinesVersion = "1.0.1"
+        coroutinesVersion = "1.1.1"
```

Comment:
😃 An inconsistent change here, the change has been updated in upstream.
I suggest a new PR as well or rebase technical❓